app/controllers/users/messages_controller.rb

```diff
@@ -70,6 +61,19 @@ def check_logged_in
     )
   end
 
+  def set_contact
+    if params[:submitted_contact_form]
+      params[:contact][:name] = @user.name
+      params[:contact][:email] = @user.email
+      @contact = ContactValidator.new(params[:contact])
+    else
+      @contact = ContactValidator.new(
+        message: '' + @recipient_user.name +
+                 _(',\n\n\n\nYours,\n\n{{user_name}}', user_name: @user.name)
```

Comment:
This is broken.

<img width="612" alt="screen shot 2018-12-10 at 18 29 39" src="https://user-images.githubusercontent.com/282788/49753079-33e3b100-fcaa-11e8-86f9-7e4ba59acdbc.png">

Reply:
Fixed in 2692084